src/h.js

```diff
@@ -22,11 +22,11 @@ export function h(tag, props) {
     }
   }
 
-  return typeof tag === "string"
+  return typeof type === "string"
     ? {
-        tag: tag,
+        type: type,
         props: props || {},
         children: children
       }
-    : tag(props, children)
+    : type(props, children)
```

Comment:
Should add `props || {}` here, but maybe in a different PR 😭 😆

Reply:
Nah might as well fix such a tiny thing while we got it (or risk forgetting...) I'll do it

Reply:
But hang on. Is there an equivalent `children || []` somewhere else?

Reply:
Oh. yeah I see it